### ISSUE TITLE
Models is blinking at the moment of first contact with the enemy

### ISSUE THREAD

**Issue by wayLANder3** [TortoiseOrHare/NoCheatZ-4#237]
You're not forced to complete all the followings, but any detail can help.

CSGO

r75 Windows

PugSetup (sourcemod)

Dedic

LAN NoSteam

donno

When i play, enemies are teleporting for me. When i scoped in AWP for example when i play on T on infenro, and scoping Middle, i  see what enemy just goes out of my scope vision. I link demo later.

Nothing

**Comment by wayLANder3** [comment]
So, there is no demo... Because anticheat don't created NoCheatZ-autorecords folder, and didn't recorded demo. This is a bug, what anticheat don't create demo folder? 
I created this folder manualy and look later, will demos record or not.

**Comment by BlackYuzia** [comment]
If demos can't save in  /csgo/autodemos/ folder
They will be save in/ csgo/ folder. Maybe.

**Comment by wayLANder3** [comment]
I know, but there is no demos in csgo directory, i search for .dem, and there is no demos there.
So, as i said - i create this folder manualy, and demos are saving. But idk why AC don't created this folder.

**Comment by wayLANder3** [comment]
[de_cache-08-16-17_06-04-43.zip](https://github.com/TortoiseOrHare/NoCheatZ-4/files/1226683/de_cache-08-16-17_06-04-43.zip)
I got now demo from my another server - DM. Watch my play (RANDOM5 wayLANdzAR-), there is some moments, i don't know how name them...

**Comment by johankh92** [comment]
Try to disable autostrafe test.

**Comment by wayLANder3** [comment]
Thanks, i'll try

**Comment by wayLANder3** [comment]
No, didn't solved situation. Player models still "teleporting".
Got a video, where you can see what i mean. When i push Mid doors on Cbble, i look for CT, but there is no CT, i check left side, but when i move crosshair to check - model is in CT down :)

https://youtu.be/ec5gE6wgKDQ

**Comment by wayLANder3** [comment]
Ok, now im find problematic module. It's wallhackblocker. I got disabled all system, and only WHblocker is enabled, but problem did not were solved. When i disabled WHBlocker - problem is gone. I enabled all systems, and disabled WHBlocker - problem is gone.

**Comment by L-EARN** [comment]
Yeah it's obvious.

Gonna remove that module completly.

**Comment by BlackYuzia** [comment]
@L-EARN maybe that offtop but.
How you can detect WH?
I mean, you can't check hi's aim to player/wall. He can look at default positions.
I think detect WH too hard.

**Comment by L-EARN** [comment]
@BlackYuzia You're right. WallhackBlocker will be removed anyway. It's already implemented in CSGO though, I don't need to waste any more time with that shit.
But others additionnal visual blockers like antiflashbang and antismoke works well so they will stay.

**Comment by wayLANder3** [comment]
@L-EARN
"WallhackBlocker will be removed anyway. It's already implemented in CSGO though"

Implemented? Where? You mean pvs_min_player_distance ? It's doesn't work.

**Comment by BlackYuzia** [comment]
@wayLANder3 I think this (pvs_min_player_distance) will work worse than the one that was before.

**Comment by T-R-K** [comment]
@wayLANder3  sv_occlude_players i think

**Comment by wayLANder3** [comment]
@T-R-K don't see any changes.. wallhack works fine with it.
